services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -378,11 +371,14 @@ def from_http_request(
         port: Optional[int],
         request_dns: Optional[str] = None,
         request_scheme: Optional[str] = None,
+        run_id: Optional[UUID] = None,
     ) -> "SchedulerData":
         dynamic_sidecar_names = DynamicSidecarNames.make(service.node_uuid)
 
         obj_dict = dict(
             service_name=dynamic_sidecar_names.service_name_dynamic_sidecar,
+            hostname=dynamic_sidecar_names.service_name_dynamic_sidecar,
```

Comment:
MINOR: This is when the name of the variables become a tongue twister :-)
How many times you read the words ``dynamic``, ``sidecar``` and ``name`` in that line :-)

Reply:
Right, making it more readable!